documentation/known-issues.md

```diff
@@ -22,3 +22,6 @@ This section lists the major known issues with Kubernetes vSphere Cloud Provider
 ## vCenter Port other than 443
 * For Kubernetes 1.6 and 1.7 releases (except Release v1.7.3 and onwards, Release v1.6.8 and onwards) vCenter Port other than 443 is not supported.
 
+## Validation with Virtual Hardware version (VM version) 13
+* Validation of VCP against Container Hosts running on VM version 13 is ongoing. [Link](https://github.com/vmware/kubernetes/issues/450)
```

Comment:
Container Hosts running on VM version 13 -> Container Hosts using Virtual Hardware version 13

Reply:
Addressed in latest commit.